### ISSUE TITLE
amount == 0 in Approve and TransferFrom

### ISSUE THREAD

**Issue by Honglei-Cong** [ONT-Avocados/python-template#5]
in OEP4Sample_raiseException.py, 

if amount == 0, can approve and transferFrom return true at the beginning?

**Comment by tonyclarking** [comment]
Yes, you can make it directly return true or false. It won't make much difference, just avoid to fire the event. You can adjust this part based on the need of your application.
